node/flows.re

```diff
@@ -412,3 +410,40 @@ let request_withdraw_proof = (state, ~hash) =>
   };
 let request_ticket_balance = (state, ~ticket, ~address) =>
   state.Node.protocol.ledger |> Ledger.balance(address, ticket);
+
+let trusted_validators_membership =
+    (~file, ~persist, state, update_state, request) => {
+  open Networking.Trusted_validators_membership_change;
+  let {signature, payload: {address, action} as payload} = request;
+  let payload_hash =
+    payload |> payload_to_yojson |> Yojson.Safe.to_string |> BLAKE2B.hash;
+  if (payload_hash |> Signature.verify(~signature)) {
```

Comment:
This if can be replaced by something similar to the one below, the main goal is to keep the guards and the flow well defined

```suggestion
  let.assert () = (
    `Failed_to_verify_payload,
    payload_hash |> Signature.verify(~signature)
  )
```